src/store-service.js

```diff
@@ -130,8 +130,8 @@
             var thisVersion = parseVersion(this.version);
             var storedVersion = parseVersion(localStorage.getItem(VERSION_KEY));
 
-            // If there's more than x.y.z, someone's probably tampered.
-            if (storedVersion.length > 3) {
+            // If there's more than x.y.z-rc.a, someone's probably tampered.
+            if (storedVersion.length > 4) {
```

Comment:
I think we should remove it, this is beyond the scope of the method 'should upgrade'. Unless there is a good reason to keep it.